src/instruments/src/EFB/index.jsx

```diff
@@ -18,9 +18,6 @@
 
 import ReactDOM from 'react-dom';
 import { renderTarget } from '../util.mjs';
+import Efb from './Efb.tsx';
 
-const Efb = () => (
-    <div />
-);
-
-ReactDOM.render(<Efb />, renderTarget);
+ReactDOM.render(<Efb currentFlight="TS516" />, renderTarget);
```

Comment:
TS516 intentional?

Reply:
It looks like some part of the shim code must have made its way to production

Reply:
It should pass the ATC FLIGHT NUMBER SimVar